bin/cylc-get-pkg-resources

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python3
+
+"""cylc [info] get-pkg-resources [OPTIONS] DIR [RESOURCES]
+
+Extract resources from the cylc.flow package, to DIR.
+
+Options:
+    --list      List available resources
+Arguments:
+    DIR         Target Directory
+    [RESOURCES] Specific resources to extract (default all).
+"""
+
+import os
+import sys
+from cylc.flow.package_resources import get_pkg_resources, list_pkg_resources
+from cylc.flow.terminal import cli_function
+
+
+@cli_function
+def main(args):
+    target_dir = args[0]
+    if len(args) > 1:
+        resources = args[1:]
+    else:
+        resources = None
+    get_pkg_resources(target_dir, resources)
+
+
+if __name__ == '__main__':
+    if set(['help', '--help', "-h"]) & set(sys.argv):
```

Comment:
Now with Python 3 we can use set literal, i.e. `{'help', '--help', '-h'}`. I actually prefer `set(['help', '--help', "-h"])` for readability, even though my IDE annoyingly suggests set/dict/list literals. 

But, NB, set/dict/list literals are also [supposedly faster](https://medium.com/@jodylecompte/dict-vs-in-python-whats-the-big-deal-anyway-73e251df8398). So in theory we could scrub a few ms (or ns?)... but again, it could be considered microbenchmarking and ignored for now :+1:

Reply:
Nice, I wasn't aware of set literals (this was my first non-trivial Python 3 PR, I think).